packages/core/strapi/src/Strapi.ts

```diff
@@ -189,6 +189,12 @@ class Strapi implements StrapiI {
 
   reload: Reloader;
 
+  future = {
+    isEnabled: (name: string) => {
+      return this.config.get('features').future[name];
+    },
+  };
```

Comment:
@alexandrebodin is that what you meant when suggesting to add it to the Strapi node class? LMK if there's a "nicer/more conventional" way to implement this 👍🏼

Reply:
Yeah basically. Ideally under a features key 1st. you can implement it like a service so it's not in the class itself. (e.g eventHub etc)